src/outputs/pipe_output.ml

```diff
@@ -314,6 +314,8 @@ object (self)
   inherit piped_output p
   inherit chan_output p
 
+  method self_sync = false
```

Comment:
Really for this one? I'd rather have a parameter of the function specifying this since we don't know what binary is going to be called.

Reply:
The process I/O are buffered internally by OCaml when calling the process creation API.

Reply:
But really that much buffered? I guess we only have pipes here (= 65 kB), and read / write should be blocking when they are full, which should be enough to sync, isn't it?

Reply:
I added the option. We should do it for `input.external` ideally one day but this is too much change for now.